lxd/daemon.go

```diff
@@ -929,6 +930,19 @@ func (d *Daemon) createCmd(restAPI *mux.Router, version string, c APIEndpoint) {
 				return response.NotImplemented(nil)
 			}
 
+			if len(action.ContentTypes) == 0 {
+				// Require application/json if not specified by handler.
+				action.ContentTypes = []string{"application/json"}
+			}
+
+			// Validate Content-Type if supplied, or if non-zero Content-Length supplied.
+			contentType, _, _ := strings.Cut(r.Header.Get("Content-Type"), ";") // Ignore multi-part boundary part.
```

Comment:
Since it's [valid](https://www.rfc-editor.org/rfc/rfc7231#section-3.1.1.1) to have white spaces around `;`, maybe we should `TrimSpaces()`?

Reply:
fixed